scan/context.go

```diff
@@ -63,12 +59,15 @@ func (s *Scanner) getContext(ctx context.Context, scContext string) (workingDir
 	}
 
 	if isSourceControlURL {
+		fmt.Println("Getting context from Git URL")
 		workingDir, err := s.getContextFromGitURL(scContext)
 		return workingDir, err
 	} else if isURL {
+		fmt.Printf("Getting context from URL %s\n", scContext)
```

Comment:
I see the URL is actually logged arround line 95 when `http.Get()` has an error. So, just want to confirm it should be ok to log this info?